Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/ContentDialogSamplePage.xaml.cs

```diff
@@ -31,8 +31,16 @@ private async void Button_Click(object sender, RoutedEventArgs e)
 			{
 				Content = "Hello world!",
 				PrimaryButtonText = "OK",
-				XamlRoot = this.XamlRoot
+				XamlRoot = this.XamlRoot,
 			};
+
+			if ((sender as Button)?.Tag?.ToString()  == "Fluent")
```

Comment:
regarding the .Tag, what you can instead do is:
```suggestion
			if (((sender as Button)?.Parent as XamlDisplay)?.UniqueKey?.Contains("_Fluent_"))
```